### DISCUSSION TITLE
Thresholding Folder not imported

### DISCUSSION THREAD

**Discussion by Az0res** [Security-Onion-Solutions/securityonion#5164] [Unsupported Versions] [answered]
Hello,

While debugging the Suricata thresholding, we noticed the pillar folder thresholding was not imported in the default file `default/pillar/top.sls`. Is this an intended behavior?
After adding the following, we were able to use it with no issues using the examples in the thresholding folder: 
```
'*_sensor':
    (...)
    - thresholding.*
```

We are aware this file will get overwritten at the next update, which is why we are wondering whats the recommended way to use the thresholding folder? Can we just copy the `default/pillar/top.sls` to `local/pillar/top.sls`?

**Comment by m0duspwnens** [comment, marked as answer]
@Az0res 
To apply the thresholding pillar, the pillar data needs to be placed in either `/opt/so/saltstack/local/pillar/global.sls` or `/opt/so/saltstack/local/pillar/minions/<MINION_ID>.sls`. If that is done, then you won't have to assign the pillar via the top file.

More details about configuring thresholding can be found in the [documention](https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=threshold#threshold).
